python/py-pytorch/files/Werror.patch

```diff
@@ -1,211 +0,0 @@
---- third_party/fbgemm/CMakeLists.txt.orig	2023-03-31 09:12:01
-+++ third_party/fbgemm/CMakeLists.txt	2023-03-31 09:14:00
-@@ -144,7 +144,6 @@
- else(MSVC)
-   string(APPEND CMAKE_CXX_FLAGS " -Wall")
-   string(APPEND CMAKE_CXX_FLAGS " -Wextra")
--  string(APPEND CMAKE_CXX_FLAGS " -Werror")
```

Comment:
this line is still present in `third_party/fbgemm/CMakeLists.txt` (I haven't checked others) - why do you think that patch is no longer needed? We do not want to compile stuff with `-Werror`.

Reply:
That patch file no longer works with the latest version. I'll have to generate a new one.